source/features/show-whitespace.css

```diff
@@ -1,6 +1,6 @@
 /*
 Can't use this because attributes are strings and calc() won't cast them
-ttps://stackoverflow.com/a/40815884
+https://stackoverflow.com/a/40815884
```

Comment:
In response to https://github.com/sindresorhus/refined-github/pull/4833#discussion_r720638114